docs/writing-tests/accessibility-testing.md

```diff
@@ -148,35 +156,61 @@ Disable accessibility testing for stories or components by adding the following
 
 <!-- prettier-ignore-end -->
 
-## Automate accessibility tests with Jest
+## Automate accessibility tests with test runner
```

Comment:
I think this section needs to mention that automated a11y testing is just one, incomplete tool for validating your components. See: https://www.smashingmagazine.com/2018/09/importance-manual-accessibility-testing/

Reply:
Of all my comments, this one would probably require the most thought. Please let me know if I can help.

Reply:
How's this:

> The most accurate way to check accessibility is manually on real devices. However, you can use automated tools to catch common accessibility issues. For example, Axe, on average, catches [57% of WCAG issues automatically](https://www.deque.com/blog/automated-testing-study-identifies-57-percent-of-digital-accessibility-issues/).
>
> These tools work by auditing the rendered DOM against heuristics based on [WCAG](https://www.w3.org/WAI/standards-guidelines/wcag/) rules and other industry-accepted best practices.

Reply:
Updated based on the comment. Let me know if both of you are ok with this. So that we can tick this item off.